### PR TITLE
Update Settings UI/UX #904

### PR DESCRIPTION
**Pull request by gibbsfromncis**
There are screenshots of changes:
https://github.com/bitshares/bitshares-ui/issues/904#issuecomment-365298989
https://github.com/bitshares/bitshares-ui/issues/904#issuecomment-365401549

Tested on MacOS 
- [x] Firefox
- [x] Chrome
- [x] Opera

Additional tests:
- [x] Tested on all media-queries
- [x] Tested on all account types (wallet-account, password-account)

### CODE REVIEW

File: app/components/Settings/Settings.jsx [hunk 1]
```diff
@@ -231,21 +253,21 @@ class Settings extends React.Component {
 
         return (
             <div className={this.props.deprecated ? "" : "grid-block page-layout"}>
```
Comment (startailcoon) [inline app/components/Settings/Settings.jsx]: To remove the blue border you should change this to `tab-layout`

`<div className={this.props.deprecated ? "" : "grid-block tab-layout"}>`

Doesn't seem to break any of your CSS as far as I can see when testing it.

**Comment by gibbsfromncis** [comment]
@svk31 should I resolve conflicts of my pull requests so you be able to merge them?

**Comment by svk31** [comment]
It would certainly be nice to not have to resolve all the various conflicts manually yea.

**Comment by gibbsfromncis** [comment]
@svk31 It was with no conflicts but after you have reverted one pull-request all brake...

**Pull request merged**